package.json

```diff
@@ -19,7 +19,7 @@
     "pack:changed": "lerna run pack --since",
     "prepublishOnly": "lerna run --stream --concurrency 1 prepublishOnly",
     "postpublish": "make publish-docs && lerna run --stream --concurrency 1 postpublish",
-    "test": "lerna run --stream --concurrency 1 --sort test"
+    "test": "lerna run --ignore sentry-browser-integration-tests --stream --concurrency 1 --sort test"
```

Comment:
```suggestion
    "test": "lerna run --ignore @sentry/browser-integration-tests --stream --concurrency 1 --sort test"
```

The name doesn't match the package name at the moment, and thus the integration tests are not ignored (and tests fail).

![image](https://user-images.githubusercontent.com/88819/145791552-f35911b6-c504-4235-85bb-63c214f04d9b.png)

https://github.com/getsentry/sentry-javascript/runs/4503737591?check_suite_focus=true#step:6:160